models/cloudtemplate_region_detail.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/container"
+)
+
+const (
+	RnCloudtemplateRegionDetail        = "regiondetail"
+	CloudtemplateRegionDetailClassName = "cloudtemplateRegionDetail"
+)
+
+type Additionalconfigforregion struct {
```

Comment:
What are our case rules? Should we use proper casing for readability or we leave as is? Then it would be something like `AdditionalConfigForRegion`.

Reply:
This applies to multiple const, type, functions, variable namings.

Reply:
And does the naming make sense? Additionalconfigforregion, was CloudTemplateRegionAttributes not an option?

Reply:
Makes sense, changed the name to CloudTemplateRegion where applicable